tests/xharness/Jenkins/Jenkins.cs

```diff
@@ -15,6 +15,36 @@
 using Xharness.Jenkins.Reports;
 
 namespace Xharness.Jenkins {
+	[Flags]
+	public enum TestSelection {
+		Mac = 0,
+		iOS32 = 1,
+		iOS64 = 2,
+		iOS = iOS32 | iOS64,
+		iOSExtensions = 8,
+		iOSMSBuild = 16,
+		tvOS = 32,
+		watchOs = 64,
```

Comment:
```suggestion
		watchOS = 64,
```

Reply:
I would apply from github, but will probably have a compilation error since is in more places, will push the change.